docs/docusaurus/docs/snippets/e2e_pandas_s3/01-create-expectations-interactively.py

```diff
@@ -0,0 +1,37 @@
+# TODO: will become from great_expectations import get_context, ExpectationSuite
+import great_expectations.expectations as gxe
+from great_expectations import get_context
+from great_expectations.core import ExpectationSuite
+from great_expectations.exceptions import DataContextError
+
+context = get_context(project_root_dir="./gx")
```

Comment:
```suggestion
context = get_context(project_root_dir=".")
```
I think the current value we have is more appropriate for a `context_root_dir`

Reply:
Same comment throughout

Reply:
Updated!